Remotion/Web/Development.WebTesting.IntegrationTests/ListMenuControlObjectTest.cs

```diff
@@ -154,6 +155,36 @@ public void TestClickItem ()
       Assert.That(home.Scope.FindId("TestOutputLabel").Text, Is.EqualTo("ItemID4|WxeFunction"));
     }
 
+    [Test]
+    public void TestHeading ()
+    {
+      var home = Start();
+
+      var listMenu = home.ListMenus().GetByLocalID("MyListMenu");
+      var heading = listMenu.Heading;
+
+      Assert.That(heading, Is.EqualTo("Test List Menu"));
+    }
+
+    [Test]
+    public void TestMissingHeading ()
+    {
+      var home = Start();
+
+      var listMenu = home.ListMenus().GetByLocalID("MyListMenu2");
+
+      var initialTimeout = listMenu.Scope.ElementFinder.Options.Timeout;
+      try
+      {
+        listMenu.Scope.ElementFinder.Options.Timeout = TimeSpan.Zero;
+        Assert.That(() => listMenu.Heading, Throws.InstanceOf<MissingHtmlException>());
```

Comment:
no heading means null/empty (check what's ususally done here) is returned. missinghtmlexception is not used when a property etc is evaluated on a controlobject.